app/src/pages/inside/historyPage/historyPage.jsx

```diff
@@ -41,8 +43,10 @@ import { HistoryView } from './historyView';
     parentItem: parentItemSelector(state),
     isTestItemsList: isTestItemsListSelector(state),
     isStepLevel: isStepLevelSelector(state),
+    includeAllLaunches: isHistoryBaseAllLaunchesSelector(state),
```

Comment:
There is no need to specifically keep it in store just for a tooltip.
You can provide this information directly via props to the required inner component.